### PR TITLE
Structurizr diagram options

### PR DESCRIPTION
**Pull request by pgrund**
- allow structurizr diagrams to be processed as well (see https://docs.kroki.io/kroki/setup/install/ for default)
- enable/pass on diagram options

### CODE REVIEW

File: kroki/plugin.py [hunk 1]
```diff
@@ -21,7 +22,7 @@
 
 class KrokiPlugin(BasePlugin):
     config_scheme = (
-        ('ServerURL', config.config_options.Type(str, default='https://kroki.io')),
+        ('ServerURL', config.config_options.Type(str, default=os.getenv('KROKI_SERVER','https://kroki.io'))),
```
Comment (b-bittner) [inline kroki/plugin.py:25]: Could you please change the OS-Var to KROKI_SERVER_URL to be more consistent with the config file

File: kroki/config.py [hunk 2]
```diff
@@ -50,4 +51,4 @@ def __init__(self, blockdiag_enabled, bpmn_enabled, excalidraw_enabled, mermaid_
 
     def get_block_regex(self, fence_prefix):
         diagram_types_re = "|".join(self.diagram_types)
-        return rf'(?:```{fence_prefix})({diagram_types_re})\n(.*?)(?:```)'
+        return rf'(?:```{fence_prefix})({diagram_types_re})((?:\s?[a-zA-Z0-9\-_]+=[a-zA-Z0-9\-_]+)*)\n(.*?)(?:```)'
```
Comment (b-bittner) [inline kroki/config.py:54]: please remove the trailing spaces

File: kroki/plugin.py [hunk 3]
```diff
@@ -85,19 +85,21 @@ def _save_kroki_image_and_get_url(self, file_name, image_data, files):
 
         return f'/{get_url}'
 
-    def _replace_kroki_block(self, match_obj, files, page):
+    def _replace_kroki_block(self, match_obj, files, page):
```
Comment (b-bittner) [inline kroki/plugin.py:89]: https://docs.kroki.io/kroki/setup/diagram-options/

**Comment by b-bittner** [comment]
Hi @pgrund ,
thanks for your PR, it looks really good. I'm going to test it and also wait for the minor changes I've requested.

**Comment by pgrund** [comment]
sorry, late summer break ... will try to update PR accordingly this week

**Review by pgrund — COMMENTED** [review]
fixed according to feedback

**Pull request merged**